src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -23,6 +24,8 @@
 using Microsoft.SqlTools.Utility;
 using System.Diagnostics;
 
+using HostingService = Microsoft.SqlTools.ServiceLayer.Hosting.ServiceHost;

```

Comment:
We were discussing that you won't need to do this, is that still the plan?

Reply:
Am I forgetting some context here - where was that being discussed?

Reply:
This was a discussion I had with @Xtrimmer. Essentially you can use the Singleton to get the Connection Service and access the ServiceHost from there.

Reply:
Oh yea, updated now.